src/pydata_sphinx_theme/assets/styles/extensions/_copybutton.scss

```diff
@@ -10,12 +10,12 @@ div.highlight button.copybtn {
   justify-content: center;
 
   // Removes the button background so we more naturally blend with the code cell.
-  background-color: unset;
```

Comment:
I wonder why the linter removed this line

Reply:
Because is it set on line 20 to `background-color: var(--pst-color-surface);` ?